content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: bt_wiki
+title: Deployments View
+category: Widgets
+draft: false
+---
+
+Presents information about deployments in a single view. Contains:
+
+- a table showing the deployments in the system (the left side) The deployments
+  can be further filtered.
+  See [Filtering deployments]({{< relref "#filtering-deployments" >}}).
+- a pane with the deployment details divided into tabs (the right side)
+- a map showing the deployments locations
+
+<!-- TODO: screenshot -->
+
+## Table
+
+The table shows deployments that match the current filter.
+See [Filtering deployments]({{< relref "#filtering-deployments" >}})
+for more details on filtering.
+
+The columns shown in the table can be shown and hidden using the widget's
+configuration.
+
+<!--  TODO: screenshot -->
+
+The columns can show:
```

Comment:
For better readability, maybe the columns description can be part of new section ("### Columns") under the "## Table" section?

Reply:
Good idea :+1: I'll reorder the paragraphs and make it a section